pmd-cli/src/test/java/net/sourceforge/pmd/cli/ForceLanguageCliTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cli;
+
+import static net.sourceforge.pmd.cli.internal.CliExitCode.OK;
+import static net.sourceforge.pmd.cli.internal.CliExitCode.VIOLATIONS_FOUND;
+import static net.sourceforge.pmd.util.CollectionUtil.listOf;
+
+import java.util.List;
+
+import org.junit.Test;
```

Comment:
```suggestion
import org.junit.jupiter.api.Test;
```

This test should be a junit5 test, because `BaseCliTest` already uses junit5 annotations